### PR TITLE
example: Train Nafnet for low-light enhancement

### PR DESCRIPTION
**Pull request by soumik12345**
Add an example for training NafNet for low-light enhancement.

### CODE REVIEW

**Review by SauravMaheshkar — DISMISSED** [review]

**Review by SauravMaheshkar — APPROVED** [review]

File: examples/low_light_enhancement/supervised/nafnet/train_nafnet.ipynb [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    "train_dataset, val_dataset = data_loader.get_datasets(batch_size=4)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = NAFNet()\n",
```
Comment (Samarendra109) [inline examples/low_light_enhancement/supervised/nafnet/train_nafnet.ipynb:49]: Just a question, we don't want to use model_configs that you defined earlier here too? They are used in the .py file.

**Review by Samarendra109 — APPROVED** [review]
LGTM to me. I just have 1 qu, but I am approving the PR. If that question doesn't require any change then feel free to submit.

File: examples/low_light_enhancement/supervised/nafnet/train_nafnet.ipynb (hunk 1 above)
Comment (soumik12345) [reply examples/low_light_enhancement/supervised/nafnet/train_nafnet.ipynb:49]: Which model configs? We're currently using the default ones.

**Review by Samarendra109 — APPROVED** [review]
LGTM

**Pull request merged**